packages/dev/core/src/FrameGraph/frameGraphRenderContext.ts

```diff
@@ -18,10 +29,15 @@ export class FrameGraphRenderContext extends FrameGraphContext {
     private _renderTargetIsBound = true;
     private readonly _copyTexture: CopyTextureToTexture;
 
+    private static _IsObjectRenderer(object: Layer | ObjectRenderer): object is ObjectRenderer {
+        return (object as ObjectRenderer).initRender !== undefined;
+    }

```

Comment:
Maybe avoid using `object` as a parameter name since it is also a keyword?
```suggestion
    private static _IsObjectRenderer(value: Layer | ObjectRenderer): value is ObjectRenderer {
        return (value as ObjectRenderer).initRender !== undefined;
    }
```